AnkiDroid/src/main/java/com/ichi2/anki/dialogs/BackupPromptDialog.kt

```diff
@@ -140,10 +140,13 @@ class BackupPromptDialog private constructor(private val windowContext: Context)
         /** @return Whether the dialog was shown */
         suspend fun showIfAvailable(deckPicker: DeckPicker): Boolean {
             val backupPrompt = BackupPromptDialog(deckPicker)
+            val isDisabledPermanently = deckPicker.sharedPrefs().getBoolean("backupPromptDisabled", false)
             if (!backupPrompt.shouldShowDialog()) {
                 return false
             }
-
+            if (isDisabledPermanently) {
```

Comment:
```suggestion
            if (deckPicker.dialogPermanentlyDismissed) {
```

Using above var definition in place of local variable / duplicated code - I *think* this will work ? Completely untested